queue/queue.c

```diff
@@ -43,7 +71,8 @@ int dequeue(Queue *q)
 */
 void destroyQueue(Queue *q)
 {
-
+	free(q->storage);
+	free(q);
 }
 
 
```

Comment:
Nice work on your queue! A lot of people had trouble with this but yours works perfectly.